ruby/BUILD.bazel

```diff
@@ -83,6 +85,35 @@ filegroup(
     ],
 )
 
+java_export(
+    name = "protobuf_java_release",
```

Comment:
nit: name this jruby instead of java?

Reply:
I was struggling with naming and ended up matching the jar name and the `protobuf_java` rule above.

I agree it would be more clear if this target had "jruby" in it, but I'm worried that makes it less clear what the other "java" means. WDYT?

Reply:
Good point, better to keep them consistent.  These will probably all get refactored soon anyway with my Bazel changes